apsuite/orbcorr/si_calc_bump_matrix_at_bc_magnet.py

```diff
@@ -28,9 +30,21 @@ def calc_matrices(minsingval=0.2, mcidx=0, deltax=10e-6):
 
     idcs = np.array(
         [bpm1_sec_index, bpm2_sec_index,
-        160+bpm1_sec_index, 160+bpm2_sec_index])
+            160+bpm1_sec_index, 160+bpm2_sec_index])
```

Comment:
This way of breaking lines is not standard. Do you have your ruff configured correctly?